dex/src/main/scala/com/wavesplatform/dex/api/ws/entities/WsMatchTransactionInfo.scala

```diff
@@ -0,0 +1,55 @@
+package com.wavesplatform.dex.api.ws.entities
+
+import com.wavesplatform.dex.api.ws._
+import com.wavesplatform.dex.domain.asset.AssetPair
+import com.wavesplatform.dex.domain.bytes.ByteStr
+import com.wavesplatform.dex.domain.model.Denormalization.{denormalizeAmountAndFee, denormalizePrice}
+import com.wavesplatform.dex.domain.order.Order
+import com.wavesplatform.dex.error.ErrorFormatterContext
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+import java.math.{BigDecimal, RoundingMode}
+
+final case class WsMatchTransactionInfo(
+  txId: ByteStr,
+  timestamp: Long,
+  price: Double,
+  executedAmountAssets: Double,
+  executedPriceAssets: Double
+)
+
+object WsMatchTransactionInfo {
+
+  def normalized(assetPair: AssetPair, txId: ByteStr, timestamp: Long, price: Long, executedAmountAssets: Long)(implicit
+    efc: ErrorFormatterContext
+  ): WsMatchTransactionInfo = {
+
+    val ad = efc.unsafeAssetDecimals(assetPair.amountAsset)
+    val pd = efc.unsafeAssetDecimals(assetPair.priceAsset)
+
+    def calcAmountOfPriceAssets(): Long = BigDecimal.valueOf(executedAmountAssets)
```

Comment:
Let's create a function`amountOfPriceAsset` in `AcceptedOrder` companion object and use it in `accepterOrder. amountOfPriceAsset` and here